app/assets/javascripts/models/ui/project_list.js.coffee

```diff
@@ -89,8 +89,18 @@ ns.ProjectList = do (ko
       @collectionsToDownload().length == 0 && @collectionOnly().length == 0 && @submittedOrders().length == 0 && @submittedServiceOrders().length == 0 && @collectionLinks().length == 0
 
     loginAndDownloadCollection: (collection) =>
-      @project.focus(collection)
-      @configureProject()
+      $('#delayOk').on 'click', =>
```

Comment:
Knockout has functionality that we've used previously for handling modals, I think? I know we have an extension it. I remember using it for the OPeNDAP stuff that I did, is there a reason that we can't use it here? (My use case didn't have multiple buttons so that may prove different)

Reply:
@abbottry - this is the only comment that I didn't make a change to address in the upcoming push (which will happen about 30 seconds after I post this comment).  

I did some searches through the code and I'm not seeing other instances to mimic; I'm not so sure there is a smoother method to attach this event handler.  I'm happy to adjust it if we can find a better way.